docs/modules/integrations/pages/zephyr-exporter.adoc

```diff
@@ -18,6 +18,10 @@ include::partial$jira-configuration.adoc[]
 |Required
 |Description
 
+|`zephyr.exporter.api-type`
+|false
+|The type of Jira API (`SCALE`, `SQUARE`). The `SCALE` type is selected by default.
```

Comment:
we need to document (or add references) what's `scale` and what's `square`, it's not transparent for users